org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/refactoring2/ReplaceFileContentChange.java

```diff
@@ -38,6 +38,13 @@ public class ReplaceFileContentChange extends ResourceChange {
 
 	private String name;
 
+	private VerifyMode verifyMode = VerifyMode.BOTH;
```

Comment:
Should this be more accessible ?

Reply:
Would having the 3 arg ctor and `getNextUndoVerifyMode()` to be protected be sufficient?

Reply:
Problem is have no time to try out what a possible subclasser would do

Reply:
Users can change the instantiation point, and can use the old ctor to get the old behavior. 
Isn't that sufficient for this PR?

Reply:
As I have said. I don’t have any time to try it actually out . Maybe someone else has

Reply:
i propose we just merge and fix when users complain....